scene/main/window.cpp

```diff
@@ -342,6 +342,25 @@ Point2i Window::get_position() const {
 	return position;
 }
 
+void Window::center() {
```

Comment:
I would find `move_to_center` more fitting as function name.
I don't like `center_to_screen`, because it would be ambiguous in the case of embedded windows.

Reply:
I don't have a strong opinion on this, if nobody else voices one I will rename it since 2 people prefer it now.